starter-code/style.css

```diff
@@ -0,0 +1,123 @@
+
+
+
+/* background colors */
+.bg-color-red {
+    background-color: #ed564b;
+}
+
+.bg-color-grey {
+    background-color: #606060;
+}
+
+.footer-dark {
+    background-color: #292929;
+}
+
+/* text color */
+.text-color-red {
+    color: #ed564b;
+}
+
+/* text styles */
+.bold {
+    font-weight: 700;
+}
+
+p {
+    font-size: 0.9em;
+}
+
+/* nav bar */
+.brand-font-size {
+    font-size: 2em;
+}
+
+.articles-container a {
+    color: #ed564b;
+}
+
+/* header */
+.read-more {
+    text-decoration: none;
+    color: white;
+    border: 1px solid white;
+    background-color: none;
+    padding: 10px 20px;
+}
+
+.carousel-item:after {
+    content:"";
+    display:block;
+    position:absolute;
+    top:0;
+    bottom:0;
+    left:0;
+    right:0;
+    background-image: url('./img/overlay.png');
+    /* background: rgba(0,0,0,0.5);  */
+  }
+
+/* .carousel-arrow {
+    width: 50px;
+    height: 100px;
+    background-color: #ed564b;
+} */
+
+.carousel-control-prev-icon, .carousel-control-next-icon {
+    padding: 40px 15px 40px 20px;
+    background-color: #ed564b;
+    margin: 0;
```

Comment:
In order to move the arrows to the sides you can use _position: absolute_.,:

`.carousel-control-prev-icon {`
  `  position: absolute;`
 `   left: 0`
}

`.carousel-control-next-icon {`
 `   position: absolute;`
`    right: 0`
`}`